admin_manual/configuration/user_provisioning_api.rst

```diff
@@ -144,7 +144,7 @@ Edits attributes related to a user. Users are able to edit email, displayname an
 **Syntax: ocs/v1.php/cloud/users/{userid}**
 
 * HTTP method: PUT
-* PUT argument: key, the field to edit (email, quota, displayname, password)
+* PUT argument: key, the field to edit (email, quota, display, password)
```

Comment:
However, not sure if we should use display here since what we really mean is "displayname". Display is just a bad choice of variable name :)curl http://admin:volkangezer@localhost/oc-git/ocs/v1.php/cloud/users